### DISCUSSION TITLE
Where to add evil-ex-define-cmd ?

### DISCUSSION THREAD

**Discussion by sebastorama** [syl20bnr/spacemacs#15883] [Q&A]
Hi, I'm trying to overwrite a couple of ex commands:

```
(evil-ex-define-cmd "vs" 'split-window-right-and-focus)
(evil-ex-define-cmd "sp" 'split-window-below-and-focus)
(evil-ex-define-cmd "term" 'vterm)
(evil-ex-define-cmd "tabn" 'tab-new)
(evil-ex-define-cmd "tabc" 'tab-close)
(evil-ex-define-cmd "tabm" 'tab-move-to)
```
It has no effect if I put these on `dotspacemacs/user-config`. However, if I eval this section *after* spacemacs has started, it works as intended. Am I doing anything wrong?

**Comment by bhw-foss** [comment]
Hey there, you can try wrapping those forms in a `with-eval-after-load' macro as a quick fix.

(with-eval-after-load "evil"
  (...))

https://www.gnu.org/software/emacs/manual/html_node/elisp/Hooks-for-Loading.html
